grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -575,7 +575,7 @@ private static Metadata generateMetadataFromThrowable(Throwable exception) {
     }
 
     @Nullable
-    static Decompressor clientDecompressor(HttpHeaders headers, DecompressorRegistry registry) {
+    public static Decompressor clientDecompressor(HttpHeaders headers, DecompressorRegistry registry) {
```

Comment:
nit: protected

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!